vendor/engines/projects/app/models/projects/ability_extension.rb

```diff
@@ -11,8 +11,17 @@ def initialize(ability)
     end
 
     def extend(user, resource)
+      # TODO: Check if we need any of these
+      #   Add ability for facility
+      #   Update condition in elsif to use Projects::Project
       if user.operator_of?(resource) && !user.facility_billing_administrator_of?(resource)
         ability.can([:create, :edit, :inactive, :index, :new, :show, :update], Projects::Project)
+      elsif SettingsHelper.feature_on?(:cross_core_order_view) && resource.is_a?(Facility) && user.facility_staff_or_manager_of_any_facility?
+        ability.can :show, Projects::Project do |project|
+          facility_ids = project.orders&.map(&:facility_id)&.uniq
```

Comment:
would it be simpler to pass  a scope here like:
`ability.can :show, Projects::Project.for_user(user)`